addons/viewport/src/Tool.tsx

```diff
@@ -70,7 +70,11 @@ interface Link extends LinkBase {
   onClick: () => void;
 }
 
-const flip = ({ width, height }: ViewportStyles) => ({ height: width, width: height });
+const flip = ({ width, height, ...styles }: ViewportStyles) => ({
+  ...styles,
```

Comment:
LGTM but it's a little strange because `ViewportStyles` only has width/height properties?!

Reply:
I added styles to remove the border from some viewports, but this didn't work anymore when I rotated the viewport. I think it would be great too to change the styles of all viewports globally.